src/components/mark/mark.tsx

```diff
@@ -27,9 +29,23 @@ export const EuiMark: FunctionComponent<EuiMarkProps> = ({
   const useTheme = useEuiTheme();
   const styles = euiMarkStyles(useTheme);
```

Comment:
Quick thought around compiled styles and the requirement for pseudo content...

Can we actually pass the i18n'd string into the `euiMarkStyles` function? So something like:

```suggestion
  const styles = euiMarkStyles({ theme: useTheme, highlightStartText: highlightStart, highlightEndText: highlightEnd });
```